cmd/openshift-install/main.go

```diff
@@ -37,8 +37,14 @@ func main() {
 	log.SetLevel(l)
 
 	assetStore := &asset.StoreImpl{}
-	if _, err := assetStore.Fetch(targetAsset); err != nil {
+	st, err := assetStore.Fetch(targetAsset)
```

Comment:
There can be multiple assets in a single target. For now it is okay.

Reply:
I think the contents of a target can just include all its assets.